big_tests/tests/gdpr_SUITE.erl

```diff
@@ -1051,18 +1089,19 @@ dont_retrieve_other_user_pubsub_payload(Config) ->
 
 retrieve_created_pubsub_nodes(Config) ->
     escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
-        Node1 = {_Domain, NodeName1} = {pubsub_tools:node_addr(), <<"node1">>},
-        Node2 = {_Domain, NodeName2} = {pubsub_tools:node_addr(), <<"node2">>},
-        Node3 = {_Domain, NodeName3} = {pubsub_tools:node_addr(), <<"node3">>},
+        [Node1={_,NodeName1}, Node2={_,NodeName2}, Node3={_,NodeName3}] =
```

Comment:
what do you think about introducing function `nodename({_, NodeName}) -> NodeName` and use it when we need `NodeName` instead of this kind of binding? So that the line above would look like:
```
[Node1, Node2, Node3] = pubsub_tools:create_node_names(3),
```

and we would have then calls like:
```
...
[pubsub_nodes_row_map(nodename(Name3), "push")]),
...
```

IMO that would increase readibility

Reply:
Don't know, let's see what others think about this one, maybe yeah